v4/feature/trade/src/main/java/exchange/dydx/trading/feature/trade/margin/components/liquidationprice/DydxAdjustMarginInputLiquidationPriceView.kt

```diff
@@ -117,24 +117,28 @@ object DydxAdjustMarginInputLiquidationPriceView : DydxComponent {
             ) {
                 Spacer(modifier = Modifier.weight(1f))
                 if (state.before != null) {
+                    val beforeTextStyle = if (state.after == null) {
```

Comment:
if there is a before and after, before text is slightly smaller according to designs.